opifancontrol.sh

```diff
@@ -102,7 +102,7 @@ smooth_ramp() {
 }
 
 while true; do
-    CPU_TEMP=$(cat /sys/class/thermal/thermal_zone0/temp)
```

Comment:
Pure optional and likely not essential: https://github.com/MichaIng/DietPi/blob/b13c6731a1d01bb3d9179f26da1cd02dea1a9b28/dietpi/func/dietpi-globals#L1721

Reply:
Thanks for thorough insight! Good to know 👍